src/shared/components/app/footer.tsx

```diff
@@ -32,6 +32,13 @@ export class Footer extends Component<FooterProps, any> {
                 {i18n.t("modlog")}
               </NavLink>
             </li>
+            {this.props.site.site_view.site.legal_information && (
+              <li className="nav-item">
+                <NavLink className="nav-link" to="/legal">
+                  {i18n.t("legal_information")}
```

Comment:
Can we just have this be `Legal` or `Privacy`? I don't want the navbars to get crowded with long text.

Reply:
This is just the translation key, made it longer for clarity. English text is "Legal".